src/inputs/input/input.jsx

```diff
@@ -87,6 +88,9 @@ const propTypes = {
      * Assign an element ID to the Input.
      */
     id: PropTypes.string,
+
+    isRedacted: PropTypes.bool,
```

Comment:
It'd be nice to put a documentation comment here, so it shows up on the UI Docs page.